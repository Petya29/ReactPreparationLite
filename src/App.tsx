import React, { useEffect } from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import AppRouter from './components/utilities/AppRouter/AppRouter';
import AppBar from './components/partials/AppBar/AppBar';
import M from 'materialize-css';
import 'materialize-css/dist/css/materialize.min.css';
import './styles/App.css';

function App() {

  useEffect(() => {
    M.AutoInit();
  }, [])

  return (
    <div className="App">
      <Router>
        <AppBar />
        <AppRouter />
      </Router>
    </div>
  );
}

export default App;
