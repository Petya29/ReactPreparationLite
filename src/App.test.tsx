import { render, screen } from '@testing-library/react';
import App from './App';

test('Renders app component', () => {
  const app = render(<App />);

  expect(app).toMatchSnapshot();
});
