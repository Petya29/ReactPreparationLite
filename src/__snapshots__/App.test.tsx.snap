// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders app component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="navBarContainer"
        >
          <nav
            class="navBar"
            style="position: fixed; display: flex; justify-content: space-around; align-items: center;"
          >
            <h4
              class="logo"
            >
              SiteName
            </h4>
            <ul
              class="menu"
            >
              <li
                class="waves-effect"
              >
                <a
                  aria-current="page"
                  class="active"
                  href="/"
                >
                  Posts
                </a>
              </li>
              <li
                class="waves-effect"
              >
                <a
                  aria-current="page"
                  class="active"
                  href="/"
                >
                  graphQL
                </a>
              </li>
            </ul>
            <i
              class="
                    large
                    material-icons
                    waves-effect
                    dropdown-trigger
                    mobileMenu
                "
              data-target="mobileMenu"
            >
              menu
            </i>
            <ul
              class="dropdown-content"
              id="mobileMenu"
            >
              <li
                class="waves-effect"
              >
                <a
                  aria-current="page"
                  class="active"
                  href="/"
                >
                  Posts
                </a>
              </li>
              <li
                class="waves-effect"
              >
                <a
                  aria-current="page"
                  class="active"
                  href="/"
                >
                  graphQL
                </a>
              </li>
            </ul>
          </nav>
        </div>
        <div
          class="home-view container"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="navBarContainer"
      >
        <nav
          class="navBar"
          style="position: fixed; display: flex; justify-content: space-around; align-items: center;"
        >
          <h4
            class="logo"
          >
            SiteName
          </h4>
          <ul
            class="menu"
          >
            <li
              class="waves-effect"
            >
              <a
                aria-current="page"
                class="active"
                href="/"
              >
                Posts
              </a>
            </li>
            <li
              class="waves-effect"
            >
              <a
                aria-current="page"
                class="active"
                href="/"
              >
                graphQL
              </a>
            </li>
          </ul>
          <i
            class="
                    large
                    material-icons
                    waves-effect
                    dropdown-trigger
                    mobileMenu
                "
            data-target="mobileMenu"
          >
            menu
          </i>
          <ul
            class="dropdown-content"
            id="mobileMenu"
          >
            <li
              class="waves-effect"
            >
              <a
                aria-current="page"
                class="active"
                href="/"
              >
                Posts
              </a>
            </li>
            <li
              class="waves-effect"
            >
              <a
                aria-current="page"
                class="active"
                href="/"
              >
                graphQL
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div
        class="home-view container"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
